src/plugins/intel_cpu/src/nodes/executors/acl/acl_ie_scheduler.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2020-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "acl_ie_scheduler.hpp"
+
+#include "arm_compute/core/CPP/ICPPKernel.h"
+#include "arm_compute/core/Error.h"
+#include "arm_compute/core/Helpers.h"
+#include <ie_parallel.hpp>
+
+namespace ov {
+namespace intel_cpu {
+
+using namespace arm_compute;
+
+ACLScheduler::ACLScheduler() = default;
+
+unsigned int ACLScheduler::num_threads() const { return parallel_get_num_threads(); }
+
+void ACLScheduler::set_num_threads(unsigned int num_threads) {}
+
+void ACLScheduler::schedule(ICPPKernel *kernel, const Hints &hints) {
+    ITensorPack tensors;
+    schedule_common(kernel, hints, kernel->window(), tensors);
+}
+
+void ACLScheduler::schedule_op(ICPPKernel *kernel, const Hints &hints, const Window &window, ITensorPack &tensors) {
+    schedule_common(kernel, hints, kernel->window(), tensors);
```

Comment:
```suggestion
    schedule_common(kernel, hints, window, tensors);
```
@allnes I think here input parameter "window" should be used instead of kernel->window(). After adding this modification, I tested "hg-s8-b1-mpii" using hello_classification and there is no accuracy issue.

Reply:
thank you, launched accuracy checker pipeline